docs/SUMMARY.md

```diff
@@ -18,6 +18,7 @@
 
 ## Demos
 
+* [Introduction and Demo Videos](demos/video-links.md)
```

Comment:
I am not sure this is where this fits best, but i cannot come up with a good alternative. Can we move this to the bottom of this list as it is the odd ball out?